webapp/channels/src/components/admin_console/filter/filter_checkbox.tsx

```diff
@@ -10,44 +10,42 @@ type Props = {
     updateOption: (checked: boolean, name: string) => void;
 };
 
-class FilterCheckbox extends React.PureComponent<Props> {
-    toggleOption = (e: React.MouseEvent) => {
+function FilterCheckbox(props: Props) {
```

Comment:
Let's expand the props here:
```suggestion
function FilterCheckbox({
    name,
    checked,
    label,
    updateOption,
} : Props) {
```